documentation/src/main/docs/kafka/health.md

```diff
@@ -1,54 +1,39 @@
 # Health reporting
 
-The Kafka connector reports the readiness and liveness of each channel
+The Kafka connector reports the startup, readiness and liveness of each channel
 managed by the connector.
 
 !!!note
     To disable health reporting, set the `health-enabled` attribute for the
     channel to `false`.
 
-## Readiness
+## Startup & Readiness
 
-On the inbound side, two strategies are available to check the readiness
-of the application. The default strategy verifies that we have at least
-one active connection with the broker. This strategy is lightweight.
+By default, both inbound and outbound channels use underlying Kafka client metrics
+to check if at least one active connection exists with the broker.
+This strategy is lightweight.
 
 You can also enable another strategy by setting the
-`health-readiness-topic-verification` attribute to `true`. In this case,
-the check verifies that:
-
--   the broker is available
-
--   the Kafka topic is created (available in the broker).
-
--   no failures have been caught
-
-With this second strategy, if you consume multiple topics using the
-`topics` attribute, the readiness check verifies that all the consumed
-topics are available. If you use a pattern (using the `pattern`
-attribute), the readiness check verifies that at least one existing
-topic matches the pattern.
-
-On the outbound side (writing records to Kafka), two strategies are also
-offered. The default strategy just verifies that the producer has at
-least one active connection with the broker.
-
-You can also enable another strategy by setting the
-`health-readiness-topic-verification` attribute to `true`. In this case,
-teh check verifies that
+`health-topic-verification-enabled` attribute to `true`.
+With this second strategy, the health checks use a Kafka Admin Client
+to access the broker. Retrieving the topics can be a lengthy
+operation. You can configure a timeout using the
+`health-topic-verification-timeout` attribute. The default timeout is set to 2
+seconds.
 
--   the broker is available
+!!!depreciation
```

Comment:
is that admonition working?

Reply:
Replaced it with `!!!warning "Deprecated"`